File: MODUL 6/PRAK602-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
jumlah_ruangan = int(input())

zetsu_putih = list(map(int, input().split()))

for i in range(jumlah_ruangan):
    jumlah_setelah_membelah = zetsu_putih[i] * (i + 1)
    print(jumlah_setelah_membelah, end=" ")

File: MODUL 6/PRAK602-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>

int main() {
    
    int jumlahRuangan;
    scanf("%d", &jumlahRuangan);

    int zetsuPutih[jumlahRuangan];
    for (int i = 0; i < jumlahRuangan; i++) {
        scanf("%d", &zetsuPutih[i]);
    }

    
        int jumlahSetelahMembelah = zetsuPutih[i] * (i + 1);
        printf("%d ", jumlahSetelahMembelah);
    }

    return 0;
}
